### ISSUE TITLE
workbench.colorCustomizations not applied to preview window?

### ISSUE THREAD

**Issue by Gruntfuggly** [Microsoft/vscode#59320]
I've changed the colour of my scrollbar with:
```
    "workbench.colorCustomizations": {
        "scrollbarSlider.activeBackground": "#f80",
        "scrollbarSlider.hoverBackground": "#b60"
    },
```
But only editor windows seem to be pick it up - when I open the preview window, it uses the normal theme colours...

**Comment by aeschli** [comment]
With 'preview window', you mean for example the markdown preview window?

**Comment by Gruntfuggly** [comment]
Yes - I wondered if it was because the preview can have it's own stylesheet? However, the scrollbar styling  looks like it is set by vscode.
